### PR TITLE
HCAP-1106 | Show last engaged by column

### PR DESCRIPTION
**Pull request by taraepp**
https://freshworks.atlassian.net/browse/HCAP-1106?atlOrigin=eyJpIjoiYjJlYmE0MDU0ZTQ4NGZiNDgwYTAzODI0NGZhZjZmODMiLCJwIjoiaiJ9

Last Engaged By column is between Last Updated and Actions columns.

Not yet sortable

### CODE REVIEW

File: client/src/constants/participantTableConstants.js [hunk 1]
```diff
@@ -162,6 +162,7 @@ const columns = {
   rosStartDate: { id: 'rosStartDate', name: 'Return of Service Start Date', sortOrder: 22 },
   rosSiteName: { id: 'rosSiteName', name: 'Site Name', sortOrder: 23 },
   employerName: { id: 'employerName', name: 'Hired By', sortOrder: 24 },
+  engagedBy: { id: 'engagedBy', name: 'Last Engaged By', sortOrder: 15.5 },
```
Comment (alva-fresh) [inline client/src/constants/participantTableConstants.js:165]: What's `sortOrder` used for? I'm curious why it's so specific to 15.5

Comment (dbayly-freshworks) [reply client/src/constants/participantTableConstants.js:165]: Sort order determines which columns to show from left to right.  We should stick to whole numbers when using it, and just shift the others down by one.

Comment (taraepp) [reply client/src/constants/participantTableConstants.js]: Good point, fixed that!

**Review by alva-fresh — APPROVED** [review]
Looks good to me!

**Pull request merged**